crates/ubrn_cli/src/android.rs

```diff
@@ -124,16 +124,10 @@ pub(crate) struct AndroidArgs {
     /// `config.yaml` file.
     ///
     /// Android:
-    ///   aarch64-linux-android,
-    ///   armv7-linux-androideabi,
-    ///   x86_64-linux-android
-    ///   i686-linux-android,
+    ///   aarch64-linux-android,armv7-linux-androideabi,x86_64-linux-android,i686-linux-android,
     ///
     /// Synonyms for:
-    ///   arm64-v8a,
-    ///   armeabi-v7a,
-    ///   x86_64,
-    ///   x86
+    ///   arm64-v8a,armeabi-v7a,x86_64,x86
```

Comment:
Small usability improvement: I found myself copy pasting this from the help text.